runelite-client/src/main/java/net/runelite/client/plugins/skybox/SkyboxPlugin.java

```diff
@@ -91,6 +103,17 @@ public void onBeforeRender(BeforeRender r)
 			return;
 		}
 
+		if
+		(
+			config.overrideMode() == SkyOverrideMode.ALL ||
+			(config.overrideMode() == SkyOverrideMode.OVERWORLD && client.getLocalPlayer().getWorldLocation().getY() < 4200)
+		)
```

Comment:
The formatting for this condition doesn't quite match what we use elsewhere in the project. I'd expect this to look like the following:

```java
if (config.overrideMode() == SkyOverrideMode.ALL
	|| (config.overrideMode() == SkyOverrideMode.OVERWORLD && client.getLocalPlayer().getWorldLocation.getY() < 4200))
{
	// ...
}
```

Reply:
Okay